### ISSUE TITLE
[ENH] Merge TsGroup?

### ISSUE THREAD

**Issue by qian-chu** [pynapple-org/pynapple#251]
In quite a few scenarios one may want to merge some TsGroup (e.g. adding new neurons, merging PETH from different runs, etc.) but currently TsGroup is non-mutable:
https://github.com/pynapple-org/pynapple/blob/ea603577767e2117c7afdaac4edf800fae4bf6ce/pynapple/core/ts_group.py#L149-L151
Maybe we can add a method that takes two TsGroups and returns a new TsGroup when the following conditions are met:
1. Non-overlapping indices
2. Same meta-info columns
3. Same time support (if none specified, infer from start and end of the merged Ts/Tsd

Happy to initiate a pull request if people see a need for this.

**Comment by gviejo** [comment]
Hi Qian Chu, I think it's doable with the condition you just listed. Maybe a class method that does something like this

```python
tsgroup = nap.TsGroup.from_group_set([tsgroup1, tsgroup2, tsgroup3])
```
Happy to review a PR.

**Comment by gviejo** [comment]
Solved in #275
